include/messages.h

```diff
@@ -117,24 +101,24 @@ struct GroupInfo {
   TLS_SERIALIZABLE(group_id,
                    epoch,
                    tree,
-                   prior_confirmed_transcript_hash,
                    confirmed_transcript_hash,
                    interim_transcript_hash,
-                   path,
                    confirmation,
                    signer_index,
                    signature);
 };
 
+enum class NodeType : uint8_t;
+
```

Comment:
am i reading it wrong, or do we have it defined twice (around line 30)

Reply:
This is a forward declaration.